### ISSUE TITLE
Hotfix branching

### ISSUE THREAD

**Issue by dextermb** [langleyfoxall/technologies#67]
### What you would like to change/add
After having a brief discussion with @wheatleyjj about hotfix branching and how people don't keep other branches sync'd we have came up with the idea for `fix/*` branches that go into `master` and quickly move up the chain (through `master`, `staging` to `production`).

While `hotfix/*` branches could still be a thing, until we get a 100% grasp on Git as a company I feel that this is a safer alternative.

### Why you would like to change/add this
While working on `yourfreebet-laravel` and `yourfreebet-yii2` we have had to put time aside for syncing up branches (more than twice?) which obviously eats into development time and increases the changes of work been overwritten.

### Checklist
- [x] I have checked that this is not a duplicate issue.

**Comment by wheatleyjj** [comment]
I think for projects that only have 1/2 contributors it's much easier to manage hotfixes - but for YFB it's almost impossible to manage.

**Comment by AlexCatch** [comment]
`hotfix/*` should only be used for emergency fixes straight into production, `fix/*` or `feature/*` should only ever go into develop

**Comment by DivineOmega** [comment]
Makes sense to me. Though, it's worth noting that this proposal makes `fix` and `feature` branches essentially the same other than the name.

**Comment by dextermb** [comment]
> Makes sense to me. Though, it's worth noting that this proposal makes `fix` and `feature` branches essentially the same other than the name.

Very true, but we did have this whole thing with wanting to name things as close to what they are. (`master` and `development` are basically the same too)
